apps/scan/frontend/src/app_root.tsx

```diff
@@ -303,16 +316,23 @@ export function AppRoot({
     );
   }
 
-  return (
-    <React.Fragment>
-      <VoterScreen
-        electionDefinition={electionDefinition}
-        systemSettings={systemSettings}
-        isTestMode={isTestMode}
-        isSoundMuted={isSoundMuted}
-        batteryIsCharging={computer.batteryIsCharging}
+  if (isCastVoteRecordSyncRequiredScreenUp) {
+    return (
+      <CastVoteRecordSyncRequiredScreen
+        returnToVoterScreen={() =>
+          setIsCastVoteRecordSyncRequiredScreenUp(false)
+        }
       />
-      <CastVoteRecordSyncModal />
-    </React.Fragment>
+    );
+  }
```

Comment:
Whereas we were previously rendering a modal on top of the `VoterScreen`, we're now not rendering the `VoterScreen` at all. The `VoterScreen` housing scanning-related hooks, so by not rendering it, we prevent scanning.